### PR TITLE
Mimir 751 scroll to result

### PR DESCRIPTION
**Pull request by Glenruben**

### CODE REVIEW

File: src/main/resources/site/i18n/phrases.properties [hunk 1]
```diff
@@ -11,6 +11,7 @@ modified = Endret
 notAvailableHeader = This page is not available in English
 notAvailableBody = For content in English,<br/>please visit our homepage
 goHome = Go to home
+close = Close
```
Comment (johnnadeluy) [inline src/main/resources/site/i18n/phrases.properties:14]: The correct phrase here would be "Lukk". This is the default phrases file!

File: src/main/resources/site/i18n/phrases_en.properties [hunk 2]
```diff
@@ -10,6 +10,7 @@ modified = Modified
 notAvailableHeader = Denne siden er ikke tilgjengelig på norsk
 notAvailableBody = Finn annet innhold på norsk<br/>via vår forside
 goHome = Gå til forsiden
+close = Lukk
```
Comment (johnnadeluy) [inline src/main/resources/site/i18n/phrases_en.properties:13]: "Close" here. This is the english phrases file. 👾 (Very easy to mix up considering the strings above)

**Review by johnnadeluy — APPROVED** [review]
Just a few teeny nitpicks on phrases! Otherwise, excellent work 🙌  
Code is clean and easy to understand as always.

File: src/main/resources/site/i18n/phrases.properties (hunk 1 above)
Comment (Glenruben) [reply src/main/resources/site/i18n/phrases.properties:14]: Oi, retta feil fil. Whoopsie!

**Pull request merged**